test/BUILD.gn

```diff
@@ -39,436 +39,496 @@ static_library("brave_test_support_unit") {
   ]
 
   public_deps = [
-    "//chrome/test:test_support",
     "//base",
     "//chrome:resources",
     "//chrome:strings",
     "//chrome/browser",
     "//chrome/common",
+    "//chrome/test:test_support",
   ]
 
   deps = [
-    "//components/gcm_driver:gcm_driver",
+    "//brave/browser/profiles:profiles",
+    "//brave/common:pref_names",
     "//components/gcm_driver:gcm_buildflags",
+    "//components/gcm_driver:gcm_driver",
+    "//mojo/core/embedder:embedder",
   ]
 
   if (is_win) {
-    deps += [
-      "//chrome/install_static/test:test_support",
-    ]
+    deps += [ "//chrome/install_static/test:test_support" ]
   }
 }
 
 if (!is_ios) {
-group("other_unit_tests") {
-  testonly = true
-  deps = [
-    "//brave/components/brave_ads/test:brave_ads_unit_tests",
-    "//brave/components/brave_rewards/test:brave_rewards_unit_tests",
-  ]
+  group("other_unit_tests") {
```

Comment:
it's pretty hard to capture meaningful changes now, so would be nice to do whitespace fixes as separate commits in the future.

Reply:
This change happened after running gn format. BTW, I'm not sure if it's better to fix such coding format fix in a separate commit. If Brave developers think it's better, I will do like that next time.

Reply:
ok, let me do it next time.